### PR TITLE
feat(readme): added unipipe-cli section on the README.md file

### PR DESCRIPTION
**Pull request by DorukAkinci**
The easiest way to deploy `Unipipe Service Broker` infrastructure and createyour first `catalog.yml` and `handler.js` files can be done with our[UniPipe-CLI](https://github.com/meshcloud/unipipe-cli) tool.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,89 @@
 # UniPipe Service Broker
 
-A universal service broker to connect CI/CD pipelines with platforms that speak [OSB API](https://www.openservicebrokerapi.org/).
+A universal service broker to connect CI/CD pipelines with platforms that speak
+[OSB API](https://www.openservicebrokerapi.org/).
```
Comment (felixzieger) [inline README.md:4]: t: Creating a separate commit for reformattings makes it easier for reviewers to review your PRs.

**Review by felixzieger — APPROVED** [review]
Having a reference under the "Configuration" section is a good start. 

Other possible sections for references: 
- How to get started
- Deployment on Azure
- How to build a CI pipeline

File: README.md (hunk 1 above)
Comment (DorukAkinci) [reply README.md:4]: i guess it is time to stop my auto-formatting settings :) i will consider that actually it is also hard to catch for me as well

File: README.md [hunk 2]
```diff
@@ -1,70 +1,89 @@
...
 
 ![A marketplace that integrates with a Service Broker](assets/OSB-Graphic-03.png "Service Broker interactions")
 
 UniPipe Service Broker was formerly known as generic OSB API.
 
 ## Configuration
 
-The custom configuration can be done via environment variables. The following properties can be configured:
+The easiest way to deploy `Unipipe Service Broker` infrastructure and create
+your first `catalog.yml` and `handler.js` files can be done with our
```
Comment (JohannesRudolph) [inline README.md:19]: handler.js is specific to using `unipipe-cli transform` - this might be a bit too early to mention here

**Review by JohannesRudolph — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (DorukAkinci) [reply README.md]: i have removed the `handler.js` part

**Pull request merged**